frontend/src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import sidebarStyles from './sidebar.module.scss';
+
+const Sidebar = () => {
+  const buttonList = [
+    { name: 'Summary' },
+    { name: 'Rota' },
+    { name: 'Employees' },
+  ];
+  return (
+    <div className={sidebarStyles.main}>
+      {buttonList.map((button) => (
+        <div key={button.name} className={sidebarStyles.menuOption}>
```

Comment:
Rather than using Divs I think it is better to use a Button element where possible - as @JAStrachan mentioned to me yesterday, as it's better accessibility. Feel free to challenge me on that one though.

After the rewrite of the class, this will fall into the `render()` method. You can either declare 3 Button elements in the html, or you can call a `{this.getButtons}` method in your class from within the render method which will return a list of elements from an array. For example:

![Screen Shot 2020-04-17 at 13 26 14](https://user-images.githubusercontent.com/39953553/79569177-1220e800-80af-11ea-877b-49234657ff4d.png)

Reply:
Slightly off-topic, but can getCardNames be written more simply as (omitting typescript fluff):
`getCardNames = () => cards.map(card => (<p>I just drew a {card.name}</p>))`
or better still
`getCardNames = cards => cards.map(card => (<p>I just drew a {card.name}</p>))`
to make the function pure?

The latter can be written even more nicely using ramda's curried map:
`getCardNames = R.map(card => (<p>I just drew a {card.name}</p>)`

Reply:
yes it can